### ISSUE TITLE
Using a garage door

### ISSUE THREAD

**Issue by masterg99** [hardillb/node-red-alexa-home-skill-web#143]
Hi,
Please can you help me with the following?
I have created my own device to control the garage door through MQTT. This all works perfectly.
I thought I would then use node-red and alexa to control it. 
Ideally it would be great to say: Alexa open garage door, close garage door or stop garage door. Those type of options don't seem availible. 

So I thought ok ill use the lock, unlock function instead.
When i say "lock garage door", it automatically closes and all works great.

When I say "unlock garage door" Alexa replies saying that I need to unlock it manually or enable voice control in the app.
When I go into the app, there is no option to enable voice control and therefore I am stuck.

Any ideas?
Thanks,
Mark

**Comment by hardillb** [comment]
Unfortunately this is a limitation of the version of the Amazon Alexa SDK version the skill is built on (and it won't be updated as that would require a full rewrite and the skill is in maintenance only mode.)

**Comment by Bra1nK** [comment]
node-red-contrib-virtual-smart-home has a garage door  type (and a lock type), the garage is supposed to be US only but I just tested it here in the UK and it works. The Alexa app on your phone will prompt you to setup a code which is then required before Alexa will open the garage door, no code is required to close it. Hope it's OK to mention another skill here but as this one is maintenance mode only I figured it would be OK.
